common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -833,6 +834,48 @@ public static int hashCodeAscii(CharSequence bytes) {
         return null;
     }
 
+    private static class Mpsc {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1118)